rollup.config.js

```diff
@@ -1,59 +1,40 @@
-
-import builtins from 'rollup-plugin-node-builtins'
-import replace from 'rollup-plugin-re'
 import resolve from 'rollup-plugin-node-resolve'
 import commonjs from 'rollup-plugin-commonjs'
-import json from 'rollup-plugin-json'
+import babel from 'rollup-plugin-babel'
+import pkg from './package.json'
 
 export default [{
     input: 'src/index.js',
     name: 'ocapi-client',
     output: {
-        file: 'dist/commercecloud-ocapi-client.js',
+        name: 'OcapiClient',
```

Comment:
What's the difference between this line and line 8?

Reply:
I checked and it seems the build is using `ocapi-client`

Reply:
Maybe it can be removed?  The rollup sample I was looking at used this key... but that sample looked a bit dated as it referenced rollup config keys that no longer exist.  I'll remove this.